site/src/comparison.rs

```diff
@@ -1487,13 +1532,14 @@ mod tests {
                 (Category::Primary, 6.0, 5.0),
             ],
             r#"
-|            | mean[^1] | max | count[^2] |
-|:----------:|:--------:|:---:|:---------:|
-| Regressions 😿 <br /> (primary) | 100.0%   | 100.0% | 1         |
-| Regressions 😿 <br /> (secondary) | N/A      | N/A | 0         |
-| Improvements 🎉 <br /> (primary) | -16.7%   | -16.7% | 1         |
-| Improvements 🎉 <br /> (secondary) | N/A      | N/A | 0         |
-| All 😿🎉 (primary) | 41.7%    | 100.0% | 2         |
+|  | count[^1] | range | mean[^2] |
```

Comment:
We should really use `https://github.com/mitsuhiko/insta` for these tests :sweat_smile: It's so annoying to change them manually.